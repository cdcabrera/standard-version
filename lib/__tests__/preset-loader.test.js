const presetLoader = require('../preset-loader');

describe('preset-loader', () => {
  it('should pass unknown presets', () => {
    expect(
      presetLoader({
        preset: 'lorem ipsum'
      })
    ).toMatchSnapshot('unknown');
  });
});
