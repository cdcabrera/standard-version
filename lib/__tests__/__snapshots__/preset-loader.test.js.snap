// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`preset-loader should pass unknown presets: unknown 1`] = `"lorem ipsum"`;
